### ISSUE TITLE
deleteUsers is not working

### ISSUE THREAD

**Issue by Curnelius** [firebase/firebase-admin-node#1029]
Hi,
Was trying for hours to delete users from a Cloud Function using the Auto API, without success.

I am using deleteUsers API from admin:

```
  admin.auth().listUsers(1000, nextPageToken).then(function(listUsersResult) {
                            listUsersResult.users.forEach(function(userRecord) {
                              var userID = userRecord.toJSON().uid;
                              usersToDelete.push(userID);
                            });
                            return;
                      }).then(() => {
                       console.log(usersToDelete.length);
                       return admin.auth().deleteUsers(usersToDelete);
                      }).catch(function(error) {
                                  console.log('Error listing users:', error);
                      });
```

This will list all users but keep showing error when deleting : 

**TypeError: admin.auth(...).deleteUsers is not a function**

Why is the deleteUsers not working ?

**Comment by hiranya911** [comment]
> TypeError: admin.auth(...).deleteUsers is not a function

You're using an old version of the SDK that doesn't have this function. You will have to upgrade.

**Comment by Curnelius** [comment]
No i don't, you don't know what version i use to be so sure.
Also, ListUsers works and they came out together.

**Comment by hiranya911** [comment]
> No i don't, you don't know what version i use to be so sure.

The error makes it pretty obvious. The version your're using simply doesn't have that method.

> Also, ListUsers works and they came out together.

No they didn't. ListUsers was released in v5.3.0. DeleteUsers was released in v8.12.0. Here are the historical release notes: https://firebase.google.com/support/release-notes/admin/node
